### PR TITLE
QA-2070 Minor fixes related to CI setup

### PR DESCRIPTION
**Pull request by uc-pperumala**
## Description
- Add IP for `qa_orange`
- If `holmes-py` is found in dir path, then return that as parent, otherwise, return the parent where `.git` is found
- Update test files where `parent_dir()` is used

## Checklist
- Run tests: `gauge run specs`
- Ensure that the tests where .tsv are uploaded are working as expected
- Ensure that the tests where `/resources` and `/download` directories are used are working

### CODE REVIEW

**Review by bpierce01 — CHANGES_REQUESTED** [review]
Tests that use files in 'resources' are not working. They are working in the main branch. I am receiving this error when trying to run any of them. The "holmes-py" is missing in the file path for grabbing resources: 

File "/Users/brentpierce/Documents/GitHub/gdc-frontend-framework/holmes-py/venv/lib/python3.10/site-packages/playwright/_impl/_set_input_files_helpers.py", line 50, in <listcomp>
      os.stat(f).st_size > SIZE_LIMIT_IN_BYTES
    FileNotFoundError: [Errno 2] No such file or directory: '/Users/brentpierce/Documents/GitHub/gdc-frontend-framework/resources/cohort_bar/organoid.tsv'

File: holmes-py/env/default/default.properties [hunk 1]
```diff
@@ -20,11 +20,12 @@ logs_directory = logs
...
 # use for testing single apps purpose in a repo
-APP_ENDPOINT_PROD = https://portal.gdc.cancer.gov/v2
+APP_ENDPOINT_PROD = https://portal.gdc.cancer.gov
 APP_ENDPOINT_LOCAL = http://localhost:3000/v2
 APP_ENDPOINT_QA_YELLOW = https://172.23.10.130/v2
 APP_ENDPOINT_QA_PINK = https://172.23.4.239/v2
 APP_ENDPOINT_QA_UAT = https://172.23.7.48/v2
+APP_ENDPOINT_QA_ORANGE = https://172.23.8.106
 APP_ENDPOINT_PROD_UAT = https://uat-portal.gdc.cancer.gov/v2
 APP_ENDPOINT_PROD_SOFT_LAUNCH = https://portal.gdc.cancer.gov
```
Comment (bpierce01) [inline holmes-py/env/default/default.properties]: We can delete "APP_ENDPOINT_PROD_SOFT_LAUNCH" now

**Review by bpierce01 — COMMENTED** [review]
Delete the "APP_ENDPOINT_PROD_SOFT_LAUNCH". I ran the code again, and the tests that use resources or downloads folder are working.

File: holmes-py/env/default/default.properties [hunk 2]
```diff
@@ -20,11 +20,12 @@ logs_directory = logs
 enable_multithreading = true
 
 # use for testing single apps purpose in a repo
-APP_ENDPOINT_PROD = https://portal.gdc.cancer.gov/v2
+APP_ENDPOINT_PROD = https://portal.gdc.cancer.gov
 APP_ENDPOINT_LOCAL = http://localhost:3000/v2
```
Comment (bpierce01) [inline holmes-py/env/default/default.properties]: This also should be: APP_ENDPOINT_LOCAL = http://localhost:3000

**Review by bpierce01 — CHANGES_REQUESTED** [review]
The readme at holmes-py/docs/test-framework.md needs to be updated as well (to reflect the updates of default.properties)

**Review by bpierce01 — APPROVED** [review]
Documentation looks good. Pulled branch and ran a couple tests and they passed. Approving

**Review by ManjunathJayanna — APPROVED** [review]
Executed tests, they are some failures, but Brent previously has noted familiarity on the same.

**Comment by uc-pperumala** [comment]
@craigrbarnes could you please merge this whenever you get a chance?

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**